### PR TITLE
Drops moped_proj_notes.added_by column

### PR DESCRIPTION
**Pull request by johnclary**
## Associated issues
- https://github.com/cityofaustin/atd-data-tech/issues/11182

## Testing
**URL to test:**  Local

**Steps to test:**
1. Start the Hasura cluster and editor

3. Create, edit, and delete comments through the three places this is possible:

- Project comments tab
- Project summary status update
- Status update modal from the Dashboard page

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

File: moped-database/metadata/tables.yaml [hunk 1]
```diff
@@ -2641,6 +2633,13 @@
           value_from_env: MOPED_API_APIKEY
         - name: MOPED_API_EVENT_NAME
           value: activity_log
+      cleanup_config:
+        batch_size: 10000
+        clean_invocation_logs: false
+        clear_older_than: 168
+        paused: true
+        schedule: 0 0 * * *
+        timeout: 60
```
Comment (johnclary) [inline moped-database/metadata/tables.yaml:2642]: Hasura did this. it's a new cloud-only feature.

**Review by chiaberry — CHANGES_REQUESTED** [review]
I can't edit or delete any of comments, which I have tracked down to 
` const editableComment =
                      userSessionData.user_id === item.added_by_user_id;` coming back as False

the COMMENTS_QUERY needs to return the added_by_user_id and then its all good

**Review by frankhereford — APPROVED** [review]
@chiaberry's comment is right on the money. I observed the same behavior, and when I added `added_by_user_id` to `COMMENT_QUERY` in `moped-editor/src/queries/comments.js#19, it worked like a charm. 

Given that change is in place, I was able to make, edit and delete comments. Thanks for this!

**Comment by johnclary** [comment]
thanks @frankhereford and @chiaberry! i completely missed this :/

taking a closer look, i see we're defaulting the `added_by_user_id` column to `1` in the DB. i believe that value would just fail in production, but in any case i don't think we want a default value for this column. i'm going to spend a little more time thinking about how this—it won't make tomorrow's release.

**Review by chiaberry — APPROVED** [review]

**Comment by johnclary** [comment]
turns out i've already broken comment editing in main (and prod?) thanks @chiaberry for patching this up!

**Pull request merged**